pkg/domain/infra/abi/play.go

```diff
@@ -640,6 +641,10 @@ func (ic *ContainerEngine) playKubePod(ctx context.Context, podName string, podY
 				if err != nil {
 					return nil, nil, err
 				}
+				// Set file permissions
+				if err := os.Chmod(f.Name(), os.FileMode(defaultMode)); err != nil {
+					return nil, nil, fmt.Errorf("cannot set permissions for file %q: %w", f.Name(), err)
```

Comment:
This will create error stutter, the os.XXX function the work on files already include the file name in the error message so you do not need to add it as well.